src/index.ts

```diff
@@ -42,10 +42,11 @@ app.set('view engine', 'pug')
 app.use(morgan('combined'))
 app.use(express.static('offsets'))
 let connectionCount = 0;
+let publicConnectionCount = process.env.PUBLIC_CONNECTION_COUNT || connectionCount;
 let address = process.env.ADDRESS;
 
 app.get('/', (_, res) => {
-	res.render('index', { connectionCount, address });
+	res.render('index', { publicConnectionCount, address });
```

Comment:
The template `views/index.pug` depends on this value. So you're breaking that template.